### PR TITLE
Set deactivation inactivity period to 7 days after notification

### PR DESCRIPTION
**Pull request by alexeykazakov**
As preparation for enabling notification & deactivation in prod.

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @alexeykazakov for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


Your plugin configuration is stored in the [file](https://github.com/fabric8-services/fabric8-auth/blob/7a371397163f3f73d26e42908e8fd9a684c5d0ef/.ike-prow/test-keeper.yaml).

File: configuration/defaults.go [hunk 1]
```diff
@@ -122,16 +122,16 @@ vwIDAQAB
...
-	defaultCheServiceURL = "http://rhche-host:8080"
 	// defaultPostDeactivationNotificationDelay the default interval between 2 cycles of the user deactivation worker
 	defaultUserDeactivationWorkerIntervalSeconds = 60 * 60 // 1 hour
 	// defaultUserDeactivationNotificationWorkerIntervalSeconds the default interval between 2 cycles of the user deactivation notification worker
 	defaultUserDeactivationNotificationWorkerIntervalSeconds = 60 * 60 // 1 hour
 	// defaultUserDeactivationRescheduleDelayDays default of 1 day to re-attempt a failed deactivation attempt
 	defaultUserDeactivationRescheduleDelayHours = 24 // 24 hours
+
+	// defaultCheServiceURL the default URL to the Che service
+	defaultCheServiceURL = "http://rhche-host:8080"
```
Comment (xcoulon) [inline configuration/defaults.go]: I believe this was not necessary (and we have something similar coming up in the e2e tests), but it's fine.

Comment (alexeykazakov) [reply configuration/defaults.go]: I didn't add this config. I just moved it a few lines above for better defaults.go organization (to keep deactivation configuration closer to each other)

Comment (xcoulon) [reply configuration/defaults.go]: ah ok, sorry. I misunderstood the change as it was presented here.

**Pull request merged**